packages/hub/.eslintrc.js

```diff
@@ -14,6 +14,9 @@ module.exports = {
       parserOptions: {
         project: './tsconfig.json',
       },
+      rules: {
+        '@typescript-eslint/no-floating-promises': 'off',
```

Comment:
tbh I'd prefer to be explicit and mark them with `void this._transport.then(...)`

Reply:
Good point, changed